File: Python/List-2/sum13.py
def sum13(nums):
  if len(nums) == 0:
    return 0
  else:
    result = 0
  for i in range(1,len(nums)):
    if nums[i-1] == 13:
      nums[i] = 0
      nums[i-1] = 0
    result += nums[i-1]
  if nums[len(nums)-1] == 13:
    return result
  return result + nums[len(nums)-1]

File: Java/Array-2/sum13.java
public int sum13(int[] nums) {
  int added = 0;
  
  if(nums.length == 0){
    return 0;
  }
  else{
    for(int i = 0; i < nums.length; i++){
      if(nums[i] != 13){
        added += nums[i];
      }
      else if(nums[i] == 13 && i < nums.length -1){
        nums[i] = 0;
        nums[i+1] = 0;
      }
    }
  }
  return added;
}

